dimod/higherorder.py

```diff
@@ -23,8 +23,92 @@
 from dimod.binary_quadratic_model import BinaryQuadraticModel
 from dimod.decorators import vartype_argument
 from dimod.vartypes import Vartype
+import dimod
+from dimod.response import SampleSet
+import numpy as np
 
-__all__ = ['make_quadratic']
+__all__ = ['make_quadratic','HigherOrderComposite']
+
+
+class HigherOrderComposite(dimod.ComposedSampler):
```

Comment:
Should consider moving this under `dimod/reference/composites`, leaving the utility functions here.